### ISSUE TITLE
OpenVR.Applications.GetApplicationKeyByIndex in Unity does not return error or app key

### ISSUE THREAD

**Issue by agoikhman** [ValveSoftware/openvr#252]

**Comment by BenWoodford** [comment]
Good to know it's not just me then.

This works perfectly fine in base C#, though the newer OpenVR headers use a StringBuilder instead of a straight string so perhaps that's the reason it doesn't work?

**Comment by agoikhman** [comment]
Yes, that’s exactly the problem, I was told an upcoming headers release would fix the declaration, but you can get it to work by modifying the header yourself.
